default.tfvars

```diff
@@ -39,6 +39,14 @@ stac_server_inputs = {
   opensearch_ebs_volume_size                  = 35
   stac_server_and_titiler_s3_arns             = []
   web_acl_id                                  = ""
+  cf_function_name                            = ""
+  cf_function_runtime                         = "cloudfront-js-2.0"
+  cf_function_code_path                       = ""
+  attach_cf_function                          = false
+  cf_function_event_type                      = "viewer-request"
+  create_cf_function                          = false
+  create_cf_basicauth_function                = false
+  cf_function_arn                             = ""
```

Comment:
similar to how `ingest = { ... }` is defined below, would it make sense to group these into a map? (i.e. `auth_function = { ... }`

The general lack of variables/re-use in TF is kind of staggering. `cf_function_name =` appears 59 times 😞

Reply:
Moved cloudfront function parameters to `auth_function = { ... }` map/object